### ISSUE TITLE
Numbered checkbox items appear in wrong order

### ISSUE THREAD

**Issue by bart-webleads** [wearebraid/vue-formulate#510]
**Describe the bug**
Numbered checkbox items appear in kind of a resorted order...

**Reproduction**
### CodePen:
https://codepen.io/bart-webleads/pen/KKvejPM

**Expected behavior**
The order for numbered items should not be changed.

**Comment by digital-codes** [comment]
Well, looking at your pen it is quite obvious: your options are {5: 'First', 2: 'Second', 3: 'Third', 4: 'Fourth'}. Ordering by key gives your output. Either you have a typo there (5 instead of 1) or you expected ordering by value instead.
I think this one could be closed.

**Comment by bart-webleads** [comment]
Yes, but why are number keys ordered, and non-numbered keys not ? This is inconsistent behavior.

**Comment by digital-codes** [comment]
Hmm, probably because numbered keys are treated like array indexes. When you modify your example to "{5: 'First', 2: 'Second', 7: 'Third', 4: 'Fourth'}" (note the 7) you get the "correct" result, assuming you sort by the key. So I still think the problem is the documentation, not the behaviour.
